splunk-otel-android/src/main/java/com/splunk/rum/Pre29ActivityCallbacks.java

```diff
@@ -119,12 +119,8 @@ public void onActivityDestroyed(@NonNull Activity activity) {
     }
 
     private ActivityTracer getTracer(Activity activity) {
-        ActivityTracer activityTracer = tracersByActivityClassName.get(activity.getClass().getName());
-        if (activityTracer == null) {
-            activityTracer = new ActivityTracer(activity, initialAppActivity, tracer, visibleScreenTracker, appStartupTimer);
-            tracersByActivityClassName.put(activity.getClass().getName(), activityTracer);
-        }
-        return activityTracer;
+        return tracersByActivityClassName.computeIfAbsent(activity.getClass().getName(),
```

Comment:
for some reason, I thought that this method wasn't Android API level 21 friendly, but I might be mis-remembering.

Reply:
:rage4: 
<img width="855" alt="image" src="https://user-images.githubusercontent.com/75337021/161641166-e9fe2363-40f7-43a9-8571-46b32b711be8.png">
It's a real bummer that there's no good way to get a warning about this in Android studio huh?

Reply:
definitely. it's very easy to get tripped up on.

Reply:
Yeah. Well, I rolled those pieces back...so worse code, better compatibility. 🤷🏻

Reply:
I have a memory of android studio complaining when I tried using unsupported stuff. This api https://developer.android.com/studio/write/java8-support-table seems to be desugared. Perhaps using it really isn't an issue?

Reply:
Turns out that `computeIfAbsent` is in the desugaring list after all!